### PR TITLE
Docker Updates

### LINKED ISSUE #3365: Docker issue
Hi,

I tried to get the Docker running locally following the instructions here https://espnet.github.io/espnet/docker.html but I get the following error:

```
Traceback (most recent call last):
  File "/miniconda/envs/espnet/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/miniconda/envs/espnet/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/espnet/espnet2/bin/asr_train.py", line 2, in <module>
    from espnet2.tasks.asr import ASRTask
  File "/espnet/espnet2/tasks/asr.py", line 51, in <module>
    from espnet2.tasks.abs_task import AbsTask
  File "/espnet/espnet2/tasks/abs_task.py", line 30, in <module>
    import wandb
ModuleNotFoundError: No module named 'wandb'
```

The steps to reproduce this issue is by running this from the espnet/docker folder 
```
./run.sh  --is-egs2 --docker-egs librispeech/asr1 --docker-folders /Datasets/LibriSpeech --ngpu 4
```
and the last commit on the local repo is e60d1786879839234cf3db8aad1f71af6f93333d

Is this a known issue? Could you help solving the issue?

Thanks for your help.

### PR DESCRIPTION
**Pull request by Fhrozen**
Fix #3365 #2738

- Updates of the docker containers, using Kaldi prebuilt binaries and with pytorch 1.9
- Modified tags to -latest
- Using Ubuntu 20.04
- Reducing storage from 25G to 14G (tag: gpu-latest)

### CODE REVIEW

File: docker/build.sh [hunk 1]
```diff
@@ -1,16 +1,18 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (sw005320) [inline docker/build.sh:1]:
```suggestion
#!/usr/bin/env bash
```

File: docker/run.sh [hunk 2]
```diff
@@ -1,13 +1,12 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (sw005320) [inline docker/run.sh:1]:
```suggestion
#!/usr/bin/env bash
```

**Review by sw005320 — APPROVED** [review]

**Comment by sw005320** [comment]
LGTM.
If it is ready, you can merge it by yourself.

**Comment by Fhrozen** [comment]
@kamo-naoyuki, could you pls merge once the checks are completed?

**Comment by sw005320** [comment]
Thanks @Fhrozen!
(You can merge docker-related PRs by yourself)

**Pull request merged**